assets/tests/hooks/useDetour.test.ts

```diff
@@ -508,18 +508,60 @@ describe("useDetour", () => {
   })
 
   describe("stops", () => {
-    test("`stops` is initially populated with the stops from the original route shape", () => {
-      const stops = stopFactory.buildList(4)
+    test("`stops` is initially populated with the stops from the original route shape with a `missed` field added", () => {
+      const stop1 = stopFactory.build()
+      const stop2 = stopFactory.build()
+
+      const { result } = renderHook(() =>
+        useDetour(
+          originalRouteFactory.build({
+            shape: shapeFactory.build({ stops: [stop1, stop2] }),
+          })
+        )
+      )
+
+      const expectedStops = [
+        { ...stop1, missed: false },
+        { ...stop2, missed: false },
+      ]
+
+      expect(result.current.stops).toStrictEqual(expectedStops)
+    })
+
+    test("when the detour is finished, `stops` excludes any missed stops", async () => {
```

Comment:
note: this test name seems like it may be out of date?

Reply:
[Woops!](https://github.com/mbta/skate/pull/2484/commits/e8905231c87f0d7ab712c1879eb244ac25c6734b)